src/archivematicaCommon/lib/storageService.py

```diff
@@ -296,20 +252,20 @@ def create_file(uuid, origin_location, origin_path, current_location,
         'related_package_uuid': related_package_uuid
     }
 
-    LOGGER.info("Creating file with {}".format(new_file))
+    LOGGER.info("Creating file with %s", new_file)
     try:
+        session = _storage_api_session(timeout=None)
```

Comment:
Possibly - but what's a sensible timeout? This is the function that store AIP calls, so it could need a very long timeout (which is why it's disabled, unlike the other calls)